scanpipe/pipes/d2d.py

```diff
@@ -774,94 +777,262 @@ def _map_javascript_resource(
             resource.update(status=flag.MAPPED)
 
 
-def _map_about_file_resource(project, about_file_resource, to_resources):
-    about_file_location = str(about_file_resource.location_path)
-    package_data = resolve.resolve_about_package(about_file_location)
+@dataclass
+class AboutFileIndexes:
+    """
+    Stores the regex path patterns, and ignore patterns,
+    package data and about file/mapped resources to create
+    packages from About files and map to deployed resources.
```

Comment:
This is a bit confusing, the summary should simply specify what that class is used for.

Reply:
Updated the docstring, thanks!